### PR TITLE
feat: Github workflow

### PR DESCRIPTION
**Pull request by andreycrane**

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+name: CI
+
+on:
+    push:
+        branches:
+            - master
+            - ci
```
Comment (Mobydack) [inline .github/workflows/CI.yml:7]: remove ci branch in branches

File: .github/workflows/CI.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+jobs:
+    test:
+        name: Test code
+
+        runs-on: ubuntu-latest
+
+        steps:
+            - uses: actions/checkout@v2
+
+            - name: Use Node.js 12 version
```
Comment (Mobydack) [inline .github/workflows/CI.yml:21]: up NodeJS version

File: .github/workflows/CI.yml (hunk 1 above)
Comment (andreycrane) [reply .github/workflows/CI.yml]: done

File: .github/workflows/CI.yml (hunk 2 above)
Comment (andreycrane) [reply .github/workflows/CI.yml]: done

**Pull request merged**